### DISCUSSION TITLE
Pyinstaller with Tensorflow-Metal

### DISCUSSION THREAD

**Discussion by odieXin** [pyinstaller/pyinstaller#7173] [Help] [answered]
I tried to package tensorflow code with pyinstaller. The CPU code path runs well, but GPU could not be enabled.
I am using Tensorflow-2.10 on Mac M1 Pro. 
My pyinstaller's version is 5.5

I installed Tensorflow and the Metal support through 
```
python3 -m pip install tensorflow-macos
pip3 install tensorflow-metal
```

I searched and found this link: https://stackoverflow.com/questions/49620638/tensorflow-gpu-with-pyinstaller
I tried suggestions listed in the above link and they did not work.

Any suggestions?

**Comment by rokm** [comment, marked as answer]
See https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/495

**Reply by odieXin** [reply]
Thank you!! @rokm
